components/dashboard/src/settings/Preferences.tsx

```diff
@@ -5,31 +5,25 @@
  */
 
 import { useContext, useState } from "react";
-import { getGitpodService } from "../service/service";
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
 import SelectableCard from "../components/SelectableCard";
-import { UserContext } from "../user-context";
-import { ThemeContext } from "../theme-context";
-import theia from '../images/theia-gray.svg';
+import Tooltip from "../components/Tooltip";
 import vscode from '../images/vscode.svg';
-import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { getGitpodService } from "../service/service";
+import { ThemeContext } from "../theme-context";
+import { UserContext } from "../user-context";
 import settingsMenu from "./settings-menu";
-import AlertBox from "../components/AlertBox";
-import Tooltip from "../components/Tooltip";
 
 type Theme = 'light' | 'dark' | 'system';
 
 export default function Preferences() {
     const { user } = useContext(UserContext);
     const { setIsDark } = useContext(ThemeContext);
-    const [defaultIde, setDefaultIde] = useState<string>(user?.additionalData?.ideSettings?.defaultIde || 'theia');
+    const [defaultIde, setDefaultIde] = useState<string>(user?.additionalData?.ideSettings?.defaultIde || 'code');
     const actuallySetDefaultIde = async (value: string) => {
         const additionalData = user?.additionalData || {};
         const settings = additionalData.ideSettings || {};
-        if (value === 'theia') {
-            delete settings.defaultIde;
-        } else {
-            settings.defaultIde = value;
-        }
+        settings.defaultIde = value;
```

Comment:
This is the reason why you wouldn't find any user with "theia" as their `additionalData.ideSettings.defaultIde` in the DB: if Theia is selected we just delete the setting (default). We should leave the if statement in and replace `'theia'` with `'code'` instead.

Reply:
I queried whether the defaultIde is not 'code' or 'code-latest' but should've queried whether they were null! Did that just now and there is plenty. Thanks for catching it. Will fix and test again.